src/syntax/rules.rkt

```diff
@@ -15,9 +17,27 @@
 
 (define *rulesets* (make-parameter '()))
 
+(define (rule-ops-supported? rule)
+  (define (ops-in-expr expr)
+    (cond
+      [(list? expr) (if (member (car expr) (*loaded-ops*))
```

Comment:
I think it's probably best to avoid `member` altogether, and use `set-member?` instead. (It has a saner return value, the name uses normal conventions, and it generalizes to hash sets if we ever want that.